packages/esm-messages-app/src/messages-admin-card-link.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Layer, ClickableTile } from '@carbon/react';
+import { ArrowRight } from '@carbon/react/icons';
+
+const MessagesManagementCardLink: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Layer>
+      <ClickableTile href={`${window.spaBase}/messages`} target="_blank" rel="noreferrer">
```

Comment:
I don't think `target="_blank"` or `noreferrer` should be used here. This is effectively navigating within the SPA, so I don't think we need a new tab, nor is there any reason to block the referrer.